chapters/4-acquisition.tex

```diff
@@ -508,12 +530,180 @@ \subsection{Using electronic survey features}
+
+\textbf{High frequency data quality checks} should be scripted 
+in advance of the start of data collection,
+so that data checks can start as soon as data starts to come in. 
+A research assistant should run the high-frequency checks (HFCs) on a daily basis 
+for the duration of the survey.\sidenote{To learn more about what types of checks should be included in the HFCs,
+	see the DIME Wiki: \url{https://dimewiki.worldbank.org/High\_Frequency\_Checks}}
+HFCs should include monitor consistency and range of responses to each question, 
+survey programming validation, tests for enumerator-specific effects, 
+and checks for duplicate entries and completeness of online submissions vis-a-vis the field log.
+
+High-frequency checks will only improve data quality
+if the issues they catch are communicated to the data provider
+and corrections are documented and applied to the data.
+This requires close communication with the field team, 
+so that enumerators are made aware of data quality issues promptly,
+and a transparent system for document issues and corrections. 
+There are many ways to communicate results of high-frequency checks to the field team.
+What's most important is to find a way to create actionable information for your team.
+\texttt{ipacheck}, for example, generates a spreadsheet with flagged errors;
+these can be sent directly to the data collection teams.
+Many teams choose other formats to display results,
+such as online dashboards created by custom scripts.
+It is also possible to automate communication of errors to the field team
+by adding scripts to link the HFCs with a messaging platform.
+Any of these solutions are possible:
+what works best for your team will depend on such factors as
+cellular networks in field work areas, whether field supervisors have access to laptops,
+internet speed, and coding skills of the team preparing the HFC workflows.
+
+Careful \textbf{field validation} is essential for high-quality survey data.
+While we cannot control natural measurement error,\sidenote{
+	See \citet{kondylis2015measuring} for an example.}
+which comes from variation in the realization of key outcomes,
+there is often an opportunity to reduce error arising from inaccuracies in the data generation process.
+\textbf{Back-checks}, spotchecks and other validation audits help ensure that 
+data is not falsified, incomplete, or otherwise suspect.\sidenote{For instructions
+	on how to design and implement Back Checks, see the DIME Wiki: \url{https://dimewiki.worldbank.org/Back\_Checks}}
+Field validation is also an opportunity to ensure that all field protocols are followed.
+For back-checks, a random subset of observations is selected,
+and a subset of information from the full survey is
+verified through a brief targeted survey with the original respondent.
+For spot-checks, field supervisors (and, if contextually appropriate, research team staff)
+should perform unannounced accompaniment of all enumerators, 
```

Comment:
"should perform unannounced accompaniment of all enumerators" - I don't understand this part of the sentence

Reply:
changed wording to clarify e87cbb9c606e7728638b96f70109b51596ac7715